extensions/wikia/AdEngine/js/lookup/rubicon/rubiconVulcan.js

```diff
@@ -148,8 +148,15 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconVulcan', [
 		log(['deleteBid', slotName, placeholderName], log.levels.debug, logGroup);
 	}
 
-	function encodeParamsForTracking(params) {
-		if (!params[rubiconVideoTierKey]) {
+	function hasResponse(slotName) {
+		var placeholderName = slotMapping[slotName],
+			vulcanResponse = win.rubiconVulcan.slots && win.rubiconVulcan.slots[placeholderName];
+
+		return vulcanResponse.rawResponses && vulcanResponse.rawResponses.length > 0;
+	}
+
+	function encodeParamsForTracking(params, slotName) {
+		if (!params[rubiconVideoTierKey] || !hasResponse(slotName)) {
```

Comment:
Just question, maybe it is worth to change condition here (and without negations)? What do you think?